pkg/codegen/python/gen_program_lower.go

```diff
@@ -54,6 +54,18 @@ func (g *generator) parseProxyApply(parameters codegen.Set, args []model.Express
 			return nil, false
 		}
 		then.Collection = arg
+
+		switch argExpr := arg.(type) {
```

Comment:
It's still not clear to me what this is fixing, can we repro this in a simpler case like l1/2-proxy-index?